### PR TITLE
Button

### PR DESCRIPTION
**Pull request by sanjitsarkar**
Added variable color

### CODE REVIEW

File: css/components/avatar/index.css [hunk 1]
```diff
@@ -1,10 +1,9 @@
 .avatar {
   border-radius: 50%;
-  border: 0.3rem solid var(--light-color);
```
Comment (sanu29) [inline css/components/avatar/index.css:3]: user px

**Review by sanu29 — APPROVED** [review]
Looks good

**Review by Ranjeetk489 — APPROVED** [review]
It looks fine now, approved

File: css/components/avatar/index.css [hunk 2]
```diff
@@ -61,25 +61,21 @@
...
-    box-shadow: var(--bx-sh-3); */
 }
 .avatar-badge {
   right: 0%;
   top: 0%;
   position: absolute;
   border-radius: 50%;
   min-width: 2rem;
-  padding: 0.25rem;
+  padding: 4px;
```
Comment (amantulsyan35) [inline css/components/avatar/index.css:78]: Below 10rem, you have two choices either to use pixels or rems, whatever you choose make it consistent through out, will make the code more uniform and consistent

**Pull request merged**